Extensions/trashbin.js

```diff
@@ -303,7 +303,7 @@
 			songs: trashSongList,
 			artists: trashArtistList
 		};
-		navigator.clipboard.writeText(JSON.stringify(data));
+		await Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
```

Comment:
Either make the function asynchronous or do this
```suggestion
		Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
```